modules/ROOT/pages/cryptography-xml.adoc

```diff
@@ -276,8 +308,9 @@ Validate an XML signed document.
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Content a| Binary |  +++the document to verify (includes the signature)+++ |  `#[payload]` | {nbsp}
-| Element Path a| String |  +++for internally detached signatures, an unambiguous XPath expression resolving to the signed element+++ |  | {nbsp}
+| Content a| Binary |  +++The document to verify (includes the signature).+++ |  `#[payload]` | {nbsp}
```

Comment:
Edit just to make the description a sentence, so that it is parallel with its peers:
```suggestion
| Content a| Binary |  +++Specifies the document to verify (includes the signature).+++ |  `#[payload]` | {nbsp}
```